mlir/lib/Target/Cpp/TranslateToCpp.cpp

```diff
@@ -1152,20 +1250,33 @@ static LogicalResult printOperation(CppEmitter &emitter,
   os << " " << functionOp.getName();
 
   os << "(";
-  Operation *operation = functionOp.getOperation();
-  if (functionOp.isExternal()) {
-    if (failed(printFunctionArgs(emitter, operation,
-                                 functionOp.getArgumentTypes())))
+
+  if (emitter.shouldPrintClass())
```

Comment:
nit: check for `!emitter.shouldPrintClass()` - because the `)` bit is common, right?